core/src/main/java/com/linecorp/armeria/common/multipart/Multipart.java

```diff
@@ -157,6 +176,32 @@ static Multipart from(String boundary, Publisher<? extends HttpData> contents, B
         return of(boundary, decoder);
     }
 
+    /**
+     * Checks whether provided {@link MediaType} is a multipart type.
+     * @param contentType {@link MediaType} to check against or {@code null} if undefined.
+     * @return true, only if provided {@link MediaType} is a multipart type.
+     */
+    static boolean isMultipart(@Nullable MediaType contentType) {
+        return contentType != null && MediaType.ANY_MULTIPART_TYPE.type().equals(contentType.type());
+    }
```

Comment:
The problem with `MediaType.is()` method is that it also uses parameters. That's why it has not been used even for `isMultipart` implementation. @trustin suggested to move it to a new class `Multiparts`. Do you agree?
If not, I can move it to `MediaType`, as you've suggested.

Reply:
I lean toward adding `isMultipart` to `MediaType`. Since `MediaType.isJson()` is added in GraphQL PR https://github.com/line/armeria/pull/3373/files#diff-9945d8d486fe454fc3bb05a2d7d8ed0194878ee5b2fb42054cb15d45b6c8c2caR887-R889.

Reply:
`MediaType.isMultipart` sounds good to me.

Reply:
Let me change to the `MediaType` then...